yucca/training/managers/alternative_trainers/YuccaManager_NoPatches.py

```diff
@@ -0,0 +1,18 @@
+# %%
+from yucca.training.managers.YuccaManager import YuccaManager
+
+
+class YuccaManager_NoPatches(YuccaManager):
```

Comment:
Just to be frank, I find this to be an abusive use of inheritance :-) I think this should be a parameter, however understand that it would screw up the `save_path`, (while we should fix that some other way....). But not sure now is the time to fix it.

![tumblr_nb6izd25Ao1tkl4vco1_500](https://github.com/Sllambias/yucca/assets/9844416/7fb0ab83-303e-4746-abaf-542798e6ee20)

Reply:
![](https://media.giphy.com/media/dIz5T9WoGagaUlJxC2/giphy.gif)